cmd/appleauth/appleauth.go

```diff
@@ -35,7 +37,7 @@ func AuthorizeWithApple(writer http.ResponseWriter, request *http.Request) {
 	initWithEnvErr := initWithEnv()
 	if initWithEnvErr != nil {
 		http.Error(writer, initWithEnvErr.Error(), http.StatusInternalServerError)
-		log.Printf("[AuthorizeWithApple] [initWithEnv]: %v", initWithEnvErr)
+		logger.LogErr(initWithEnvErr, "initWithEnv", nil)
```

Comment:
So I think in the `LogErr` method in `sawmill`, we should change the param order to be:

`operation, error, request`

Thoughts on this? I feel like there is a lot more readability in this type of layout.

Reply:
[nit]

`initWithEnv` -> `InitWithEnv`

Reply:
Sure! If that looks order looks better to you I'm glad to implement it.